api/src/navLinks/dto/create-navLinks.dto.ts

```diff
@@ -0,0 +1,5 @@
+
+export class CreateLinksDto {
```

Comment:
why do you name this `create`LinkeDto? can't we name it linksDto? I am asking real question not requesting you to change it.

Reply:
cuz i understood that the dto is to create a doc in a specific type ?
but as i'm learning r8 nw that thought was wrong.

Reply:
DTo gives us the upper hand to decide what data-schema can the client-side and the server-side can exchange in a single request/response cycle